src/zos_files/zowe/zos_files_for_zowe_sdk/datasets.py

```diff
@@ -526,30 +655,62 @@ def write(self, dataset_name, data, encoding=_ZOWE_FILES_DEFAULT_ENCODING):
         response_json = self.request_handler.perform_request("PUT", custom_args, expected_code=[204, 201])
         return response_json
 
-    def download(self, dataset_name, output_file):
-        """Retrieve the contents of a dataset and saves it to a given file."""
+    def download(self, dataset_name: str, output_file: str) -> None:
+        """
+        Retrieve the contents of a dataset and saves it to a given file.
+
+        Parameters
+        ----------
+        dataset_name: str
+            Name of the dataset to be downloaded
+        output_file: str
+            Name of the file to be saved locally
+        """
         response = self.get_content(dataset_name, stream=True)
         with open(output_file, "w", encoding="utf-8") as f:
             for chunk in response.iter_content(chunk_size=4096, decode_unicode=True):
                 f.write(chunk)
 
-    def download_binary(self, dataset_name, output_file, with_prefixes=False):
-        """Retrieve the contents of a binary dataset and saves it to a given file.
+    def download_binary(self, dataset_name: str, output_file: str, with_prefixes: bool = False) -> None:
+        """
+        Retrieve the contents of a binary dataset and saves it to a given file.
 
         Parameters
         ----------
-        dataset_name:str - Name of the dataset to download
-        output_file:str - Name of the local file to create
-        with_prefixes:boolean - If true, include a four big endian bytes record length prefix.
-                                The default is False
+        dataset_name : str
+            Name of the dataset to download
+        output_file : str
+            Name of the local file to create
+        with_prefixes : bool
+            If true, include a four big endian bytes record length prefix. The default is False
```

Comment:
Should we use formatting consistent with `get_binary_content`?
```suggestion
            If true, include a four big endian bytes record length prefix. Default: False
```